R/msa.R

```diff
@@ -196,7 +196,7 @@ createMSA_PDF <- function(fasta_path, out_path = NULL,
 #' will be saved.
 #'
 #' @importFrom Biostrings readAAStringSet
-#'
+#' @importFrom stats kalign
```

Comment:
The stats package does not export `kalign()`, but looking at the nearby code, we may need to incorporate some code from here: https://github.com/mhahsler/rMSA/blob/master/R/kalign.R

Reply:
This is likely outside the scope of this PR/issue, but a great find!!